libr/include/r_anal.h

```diff
@@ -300,6 +300,7 @@ typedef struct r_anal_function_t {
 	RBNode rb;
 	RBNode addr_rb;
 	RList *imports; // maybe bound to class?
+	bool is_noreturn;
```

Comment:
can you put it after `bp_frame` with all other bools? Also, a short comment would be great

Reply:
ok

Reply:
done